docs/usage.md

```diff
@@ -428,6 +428,13 @@ process {
 >
 > If you get a warning suggesting that the process selector isn't recognised check that the process name has been specified correctly.
 
+## Azure Resource Requests
+
+To be used with the `azurebatch` profile by specifying the `-profile azurebatch`.
```

Comment:
I think this can go to the template code? this seems relevant for all pipelines

Reply:
https://github.com/nf-core/tools/pull/1800